community/modules/scripts/pbspro-preinstall/variables.tf

```diff
@@ -29,6 +29,15 @@ variable "region" {
   type        = string
 }
 
+variable "devel_rpm" {
```

Comment:
It looks like this is going to be a requirement going forward as written. Would it make sense to make this optional, in case the user is not using spack?

I'll defer to your knowledge on this; I'm familiar enough with PBS to know if that is a reasonable requirement.

Reply:
Offline conversation: this RPM is included with standard tarball from Altair. It is reasonable to require uploading it to bucket. Separate question whether and where to require installing it.